jdbc/src/main/java/com/yahoo/ycsb/db/JdbcDBClient.java

```diff
@@ -474,7 +486,21 @@ public Status insert(String tableName, String key, HashMap<String, ByteIterator>
       for (String value: fieldInfo.getFieldValues()) {
         insertStatement.setString(index++, value);
       }
-      int result = insertStatement.executeUpdate();
+      int result;
+      if (batchSize > 0) {
+        insertStatement.addBatch();
+        if (++numRowsInBatch % batchSize == 0) {
+          int[] results = insertStatement.executeBatch();
```

Comment:
we should also do this in cleanup, if there are outstanding batched inserts. otherwise we have to rely on the record count for insert being an even multiple of the batch size, I think?